pkg/postgres/pgtest/conn.go

```diff
@@ -19,3 +19,14 @@ func GetConnectionString(_ *testing.T) string {
 	host := env.GetString("POSTGRES_HOST", "localhost")
 	return fmt.Sprintf("host=%s port=5432 database=%s user=%s password=%s sslmode=disable statement_timeout=600000 pool_min_conns=1 pool_max_conns=90", host, database, user, pass)
 }
+
+// GetGormConnectionString returns a connection string for Gorm DB
+func GetGormConnectionString(_ *testing.T) string {
```

Comment:
Why not use the same one?

Reply:
pool_min_conns=1 pool_max_conns=90 are the pool connection config, it is not supported by gorm.

Reply:
It's also not supported by pg 14 I think it's safe to remove it